cms/envs/devstack.py

```diff
@@ -140,6 +140,7 @@ def should_show_debug_toolbar(request):
     'JWT_SECRET_KEY': 'lms-secret',
     'JWT_ISSUER': 'http://127.0.0.1:8000/oauth2',
     'JWT_AUDIENCE': 'lms-key',
+    'JWT_AUTH_HEADER_PREFIX': 'JWT',
```

Comment:
no need for those. As this updates the JWT_AUTH of base . its already there. just make sure we have blacklist app in the installed apps.